### ISSUE TITLE
AttributeError: module 'attr' has no attribute 's'

### ISSUE THREAD

**Issue by bhaskoro-muthohar** [pandas-profiling/pandas-profiling#431]
**Describe the bug**

Hi, I tried to run ProfileReport but raised an error

**To Reproduce**

```python
from pandas_profiling import ProfileReport

prof = ProfileReport(df1)
prof.to_file(output_file='output.html')
```

_Data:_
My  Dataframe from a CSV which looks like this:
```
booking_date,city,Count
14-12-17, Balikpapan,2
14-12-17, Bandar Lampung,7
14-12-17, Bandung,71
14-12-17, Banjarmasin,2
14-12-17, Bantul,1
14-12-17, Batam,2
...
```

_Error:_ :
```python
HBox(children=(FloatProgress(value=0.0, description='variables', max=3.0, style=ProgressStyle(description_widt…

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-9-ed9145cd6668> in <module>
      1 from pandas_profiling import ProfileReport
      2 
----> 3 prof = ProfileReport(df1)
      4 prof.to_file(output_file='output.html')

~\AppData\Roaming\Python\Python37\site-packages\pandas_profiling\__init__.py in __init__(self, df, minimal, config_file, **kwargs)

~\AppData\Roaming\Python\Python37\site-packages\pandas_profiling\model\describe.py in describe(df)

~\anaconda3\lib\multiprocessing\pool.py in next(self, timeout)
    746         if success:
    747             return value
--> 748         raise value
    749 
    750     __next__ = next                    # XXX

~\anaconda3\lib\multiprocessing\pool.py in worker(inqueue, outqueue, initializer, initargs, maxtasks, wrap_exception)
    119         job, i, func, args, kwds = task
    120         try:
--> 121             result = (True, func(*args, **kwds))
    122         except Exception as e:
    123             if wrap_exception and func is not _helper_reraises_exception:

~\AppData\Roaming\Python\Python37\site-packages\pandas_profiling\model\describe.py in multiprocess_1d(args)

~\AppData\Roaming\Python\Python37\site-packages\pandas_profiling\model\describe.py in describe_1d(series)

~\AppData\Roaming\Python\Python37\site-packages\pandas_profiling\model\describe.py in describe_categorical_1d(series, series_description)

~\AppData\Roaming\Python\Python37\site-packages\visions\__init__.py in <module>
----> 1 from visions import core
      2 from visions import utils
      3 
      4 from visions.core.dtypes.boolean import BoolDtype
      5 from visions.core.implementations.types import *

~\AppData\Roaming\Python\Python37\site-packages\visions\core\__init__.py in <module>
----> 1 from visions.core import model
      2 from visions.core import implementations
      3 from visions.core.functional import (
      4     type_cast_frame,
      5     type_inference_frame,

~\AppData\Roaming\Python\Python37\site-packages\visions\core\model\__init__.py in <module>
----> 1 from visions.core.model.type import VisionsBaseType, TypeRelation
      2 from visions.core.model import typeset
      3 from visions.core.model.typeset import VisionsTypeset

~\AppData\Roaming\Python\Python37\site-packages\visions\core\model\type.py in <module>
     38 
     39 
---> 40 @attr.s(frozen=True)
     41 class TypeRelation:
     42     """Relationship encoder between implementations of :class:`visions.core.models.VisionsBaseType`

AttributeError: module 'attr' has no attribute 's'
```
--> 

**Version information:**

<!--
Version information is essential in reproducing and resolving bugs. Please report:

* _Python version_: Python 3.7.6
* _Environment_: Jupyter Lab local

<details><summary>Click to expand <strong><em>Version information</em></strong></summary>
<p>

```
<<< Put your version information here >>>
```

</p>
</details>

**Comment by sueny** [comment]
Try

` pip uninstall attr `
` pip install attrs`

**Comment by titobrezende** [comment]
Same here.   pip uninstall attr and install  did not work.  (Using windows with anaconda and jupyter notebook)

**Comment by bhaskoro-muthohar** [comment]
> Try
> 
> `pip uninstall attr`
> ` pip install attrs`

Did not work for me

**Comment by sbrugman** [comment]
It's an issue with one of our dependencies. I'll make a new release, that should fix it. It might take a few days before the conda forge version is up-to-date too.

**Comment by Juphex** [comment]
changing from attr to attrs worked for me

**Comment by Aicha-cher** [comment]
didn't work for me

**Comment by sidvish24** [comment]
Along with uninstalling attr package, using the minimal = True option worked for me.

"profile = ProfileReport(large_dataset, minimal=True)"

**Comment by Ajith8** [comment]
Tried with
!pip uninstall attr
!pip install attrs
Now am unable to launch Notebook as well.

Getting,

Traceback (most recent call last):
File "C:\Users\Harish\anaconda3\Scripts\jupyter-notebook-script.py", line 6, in 
from notebook.notebookapp import main
File "C:\Users\Harish\anaconda3\lib\site-packages\notebook\notebookapp.py", line 80, in 
from .services.contents.manager import ContentsManager
File "C:\Users\Harish\anaconda3\lib\site-packages\notebook\services\contents\manager.py", line 17, in 
from nbformat import sign, validate as validate_nb, ValidationError
File "C:\Users\Harish\anaconda3\lib\site-packages\nbformat\__init__.py", line 32, in 
from .validator import validate, ValidationError
File "C:\Users\Harish\anaconda3\lib\site-packages\nbformat\validator.py", line 12, in 
from jsonschema import ValidationError
File "C:\Users\Harish\anaconda3\lib\site-packages\jsonschema\__init__.py", line 11, in 
from jsonschema.exceptions import (
File "C:\Users\Harish\anaconda3\lib\site-packages\jsonschema\exceptions.py", line 152, in 
@attr.s(hash=True)
AttributeError: module 'attr' has no attribute 's'

**Comment by Aicha-cher** [comment]
> Along with uninstalling attr package, using the minimal = True option worked for me.
> 
> "profile = ProfileReport(large_dataset, minimal=True)"

Thanks worked for me too

**Comment by sbrugman** [comment]
Should be fixed in the v2.6.0 release.
